data_steward/tools/load_submission.py

```diff
@@ -0,0 +1,124 @@
+    for blob in blobs:
+        table_name = _filename_to_table_name(blob.name)
+        if table_name not in AOU_REQUIRED:
+            continue
+        schema = get_table_schema(table_name)
+        hpo_table_name = f'{hpo_id}_{_filename_to_table_name(blob.name)}'
+        fq_hpo_table = f'{bq_client.project}.{dst_dataset}.{hpo_table_name}'
+        destination = Table(fq_hpo_table, schema=schema)
+        destination = bq_client.create_table(destination, exists_ok=True)
+        job_config = LoadJobConfig()
+        job_config.schema = schema
+        job_config.skip_leading_rows = 1
+        job_config.source_format = 'CSV'
+        source_uri = f'gs://{bucket_name}/{blob.name}'
+        load_job = bq_client.load_table_from_uri(source_uri,
+                                                 destination,
+                                                 job_config=job_config)
+        LOGGER.info(f'table:{destination} job_id:{load_job.job_id}')
+        load_jobs.append(load_job)
+        load_job.result()
+    return load_jobs
+
+
+def get_arg_parser():
+    argument_parser = argparse.ArgumentParser(description=__doc__)
+    argument_parser.add_argument(
+        '-p',
+        '--project_id',
+        dest='project_id',
+        action='store',
+        help='Identifies the project containing the target dataset',
+        required=True)
+    argument_parser.add_argument('-d',
+                                 '--dataset_id',
+                                 dest='dataset_id',
+                                 action='store',
+                                 help='Identifies the target dataset',
+                                 required=True)
+    argument_parser.add_argument('-b',
+                                 '--bucket_name',
+                                 dest='bucket_name',
+                                 action='store',
+                                 help='Bucket containing archive folders',
+                                 required=True)
+    argument_parser.add_argument('-i',
+                                 '--hpo_id',
+                                 dest='hpo_id',
+                                 action='store',
+                                 help='Identifies the hpo_id of the site',
+                                 required=True)
+    argument_parser.add_argument('-f',
+                                 '--folder_name',
+                                 dest='folder_name',
+                                 action='store',
+                                 help='Name of the submission folder to load',
+                                 required=True)
+    return argument_parser
+
+
+def main(project_id, dataset_id, bucket_name, hpo_id, folder_name):
+    bq_client = get_client(project_id)
+    gcs_client = Client(project_id)
+    site_bucket = get_bucket(bq_client, hpo_id)
+    prefix = f'{hpo_id}/{site_bucket}/{folder_name}'
+    load_stage(dataset_id, bq_client, bucket_name, prefix, gcs_client, hpo_id)
```

Comment:
Is there a reason `load_stage` is returning an object but the script isn't using the returned object?

Reply:
I wasn't sure what to do with the list of job_ids. They are logged inside load_stage, but let me know if you have ideas on using them here.

Reply:
I think the `load_stage` function is good.  I just don't think we need to return a value if there are no plans to use it.  Since the job_ids are not needed in this script, do you think `load_stage` will/should ever be used outside this module allowing a different script to run this function and get the job_ids from it?  Or would you prefer if a different script ran this one to funnel it through the `main`?

I think what to do depends on other possible uses.

Reply:
I can see this being used to load other files from the archive too, but they may not have the prefix similar to the site files (e.g. vocabulary). I think in such cases the `load_stage` function needs to be used directly to allow for that flexibility.

Reply:
Alright.  In the interest of python's "Explicit is better than implicit", would you consider using
```
_ = load_stage(...)
```